### ISSUE TITLE
Front page recent discussions not updating

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/hernehill.org.uk#631]
new site
logged on as admin

There have been new posts on the Forum but these are not being updated on the front page links:

![image](https://cloud.githubusercontent.com/assets/6060985/7022127/7f86a318-dd20-11e4-94d5-0a2c56698221.png)

**Comment by pmackay** [comment]
This is actually by intent - the view only shows comments, which are
replies to posts.

**Comment by GilesGibson** [comment]
Ah, so a new post doesn't show up? This isn't going to promote the Forum to people, can this be changed so that the front page reflects the latest content?

**Comment by pmackay** [comment]
Would it be ok to defer this to an update to the new forum software?

**Comment by GilesGibson** [comment]
If the new forum software isn't too far off then fine. If it is a quick fix then it would help promote the forum conversations.

**Comment by pmackay** [comment]
Very basic new forum topic list added to frontpage. Will theme it a bit later.

**Comment by GilesGibson** [comment]
OK, we now have two areas showing roughly the same things.... recent discussions and recent topics. Could we just have the one in the centre that shows all the recent posts? @djwesto may have a view.

**Comment by pmackay** [comment]
As mentioned before I built the 2nd view, this is a limitation of Drupal
Forum, in that it uses nodes for the 1st topic and comments for the
replies. Not easy to combine.

**Comment by GilesGibson** [comment]
It is just that we are now duplicating stuff on the front page. May be worth replacing the central one with the one on the right hand side? Be great if we could add in new events that have been created, new service listings. and so on. The problem of just listing the comment is that it makes it very disjointed. I realise that this is a struggle bending Drupal to get it to behave the way we want.

**Comment by pmackay** [comment]
Its not duplicating, they just look similar. But you asked for recent new
topics?

New events, service listings is totally different. Plus there are events in
left col.

**Comment by GilesGibson** [comment]
I suppose what we are after is all the recently added content, whatever it may be. Left hand column only shows the events that are imminent. Although we are getting feedback about people not seeing their event, simply because there are too many others in front of theirs. 

What we now have is the forum topic on the right and the comment that answers it in the centre. Will Harmony get round this?

**Comment by pmackay** [comment]
Yes Harmony would help with showing forum content in a better way.

**Comment by djwesto** [comment]
I'm getting the impression from @pmackay that it's impossible to combine
forum topics (new forum posts) with a view of forum comments (replies) in
the same listing.

That's a real pain but I think it's an acceptable compromise until we move
to Harmony Forum. At that point we should review exactly how the home page
works as j would definitely want to see some sort of combined listing with
everything in one list. I'd also want to see comments differentiated from
new topic posts with a visual indication – perhaps an icon or a
different design for replies.

We would need to design this with you and Tori, @GilesGibson. I still have
a dream of a more Facebook-style activity stream which reports everything
that's happened on the site, irrespective of whether it's a new Group
that's been added, a trader promotion or new service that's been started.
All in one stream.

On Sunday, 12 April 2015, Paul Mackay notifications@github.com wrote:

> Its not duplicating, they just look similar. But you asked for recent new
> topics?
> 
> New events, service listings is totally different. Plus there are events in
> left col.
> 
> On Sun, Apr 12, 2015 at 10:26 AM, GilesGibson <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>>
> wrote:
> 
> > It is just that we are now duplicating stuff on the front page. May be
> > worth replacing the central one with the one on the right hand side? Be
> > great if we could add in new events that have been created, new service
> > listings. and so on. The problem of just listing the comment is that it
> > makes it very disjointed. I realise that this is a struggle bending
> > Drupal
> > to get it to behave the way we want.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/folklabs/hernehill.org.uk/issues/631#issuecomment-92022306
> > 
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/folklabs/hernehill.org.uk/issues/631#issuecomment-92023075
> .

## 

---

_Chris Wells_\* Co-founder & Creative Director*
Folk Labs Ltd           www.folklabs.com           linkedin
http://lnkd.in/uZxawZ          twitter @djwesto
http://www.twitter.com/djwesto          mobile +44 (0)7786 221 654

**Comment by GilesGibson** [comment]
@djwesto The combined stream of everything that is happening on the site would be the aim. It triggers interest as at the moment it is hard to find out what may have changed, what new events, new postings in a group etc. I really think that splitting topics from comments in Forum is strange, I know there are technical reasons but the end result on the front page is really confusing.
